tests/test_headers.py

```diff
@@ -1,55 +1,40 @@
 # -*- coding: utf-8 -*-
 
 
-def test_id(headers):
-    assert 'X-GitHub-Request-Id' in headers
-
-
 def test_date(headers):
     assert 'Date' in headers
 
 
-def test_expires(headers):
-    assert 'Expires' in headers
-
-
 def test_content_encoding(headers, case):
```

Comment:
Tricky. Use case as dependency to response and response as dependency to headers and use case as single fixture. https://www.python.org/dev/peps/pep-0020/#id3 - "Explicit is better than implicit."